macros/edr/tests/test_all_columns_anomalies.sql

```diff
@@ -61,14 +59,17 @@
                 {%- set ignore_column = elementary.should_ignore_column(column_name, exclude_regexp, exclude_prefix) -%}
                 {%- if not ignore_column -%}
                     {%- do monitors.extend(column_monitors) -%}
-                    {%- set min_bucket_start = elementary.edr_quote(elementary.get_test_min_bucket_start(model_graph_node,
-                                                                                                     backfill_days,
-                                                                                                     column_monitors,
-                                                                                                     column_name,
-                                                                                                     metric_properties=metric_properties)) %}
+                    {% if timestamp_column and is_timestamp %}
```

Comment:
We assert in the upper rows that if a timestamp column is given then `is_timestamp` is True
so I think a cleaner look can be:
```suggestion
                    {% if timestamp_column %}
```